### DISCUSSION TITLE
Lidl HG06338 behavior in a group

### DISCUSSION THREAD

**Discussion by Nardol** [Koenkk/zigbee2mqtt#12824] [Devices]
Hello,

I have a Lidl [HG06338](https://www.zigbee2mqtt.io/devices/HG06338.html#lidl-hg06338) 4 gangs switch with 4 USB, the old one.

I have added it to a group to turn all switches on/off.

I understood if I don't specify the endpoint, all should be concerned.
Firstly, am I wrong? :slightly_smiling_face:

If no, in this case I observe something strange: if I put the device in the group and switch the group off when all switches are on, only l1 is switched off. I would expect all switches turning off.
For now as a workaround solution I added the three endpoints in the group but I would like to understand.

I hop I gave enough information to have an answer :slightly_smiling_face:

**Comment by Koenkk** [comment]
You need to add all endpoints to the group
